modules/azure-oracle-adbs/README.md

```diff
@@ -0,0 +1,57 @@
+# azure-oracle-adbs
+
+## Summary
+
+Terraform module for **Oracle Autonomous Database Service (ADBS)** , Creates a new Autonomous Database resource in Oracle Database @ Azure.
+
+## Pre-requisites
+
+A virtual network with subnet delegated to `Oracle.Database/networkAttachment` present in targetted subscription.
+
+## Providers
+
+| Name                                                   | Version |
+| ------------------------------------------------------ | ------- |
+| [azapi](#provider_azapi) | n/a     |
```

Comment:
We are using azapi here

Reply:
yes we are using `azapi_resource` for creating autonomous_db resource.